src/main/java/seedu/condonery/model/property/Price.java

```diff
@@ -0,0 +1,59 @@
+package seedu.condonery.model.property;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.condonery.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Property's price in Condonery.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPrice(String)}
+ */
+public class Price {
+
+    public static final String MESSAGE_CONSTRAINTS = "Prices can take any positive integer, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "^\\d+([,]\\d+)*$";
+
+    public final String value;
```

Comment:
If price is string do we need to always convert back to `int` when performing manipulations such as below or above a certain price? Would `int` type be better?